content/francais/python-turtle/activity-7.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Activité 7 - Dessiner un chiffre 8"
+date: 2024-09-17T13:24:17-07:00
+weight: 10
+draft: false
+---
+
+<img src="../media/turtle_circles.png" alt="dessiner un chiffre 8" width="25%"/>
+
+Super ! Maintenant, faisons un chiffre 8 comme première étape pour créer une fleur mandala ! Un chiffre 8 est simplement deux cercles opposés l'un à l'autre. 
+Pour ce faire, nous pouvons utiliser la méthode `turtle.circle(x)`, qui prend un entier `x` comme rayon du cercle. (Vous vous souvenez peut-être que `2 * rayon` est la largeur du cercle.) Avoir le rayon comme **argument** signifie que nous pouvons décider de la taille du cercle.
```

Comment:
In sentence: " (Vous vous souvenez peut-être que `2 * **rayon**` est la largeur du cercle.)"
radius got translated even if is inside backticks, and other inline code did not got translated. Not sure if we want to keep it in English or Spanish, but why did this instance got translated?